src/PointwiseFunctions/AnalyticSolutions/RelativisticEuler/TovStar.cpp

```diff
@@ -284,6 +284,14 @@ TovStar<RadialSolution>::variables(
   return make_with_value<tnsr::ii<DataType, 3>>(x, 0.0);
 }
 
+template <typename RadialSolution>
+template <typename DataType, typename Tag>
+tuples::TaggedTuple<::Tags::dt<Tag>> TovStar<RadialSolution>::variables(
```

Comment:
for my understanding, the `dt` is needed for constructing `Pi`, right?

Reply:
Yeah, that's right; internally, the `WrappedGr` grabs `dt_shift`, `dt_lapse, and dt_spatial_metric and assembles `Pi`.